cypress/support/step_definition/rooms/commonRoomsSteps.spec.js

```diff
@@ -3,4 +3,17 @@ import Rooms from "../../pages/rooms/pageRooms";
 
 const rooms = new Rooms();
 
-// this common step definition file can be used to have all common rooms related step definitions in one place
+When("I click on FAB to add participants", () => {
+	rooms.clickOnAddParticipantsFAB();
+});
+
+Then(
+	"I see speed dial options {string} and {string}",
+	(selectFromDirectory, addExternalPerson) => {
+		rooms.seeSpeedDialOptions(selectFromDirectory, addExternalPerson);
```

Comment:
I would suggest to reflect the flexibility in the naming of the variables in this way:
```suggestion
	(...options) => {
		rooms.seeSpeedDialOptions(options);
```